package.json

```diff
@@ -6,15 +6,16 @@
     "ts-mocha": "^10.0.0",
     "ts-proto": "^1.143.0",
     "typedoc": "^0.24.7",
-    "typescript": "^5.0.2"
+    "typescript": "^5.0.2",
+    "c8": "^8.0.1"
   },
   "workspaces": [
     "./packages/*"
   ],
   "scripts": {
     "docs": "typedoc",
-    "test": "npm run test --workspaces",
-    "test-except-broken-links": "npm run test --workspace=@google-cloud/synthetics-sdk-api && npm run test --workspace=@google-cloud/synthetics-sdk-mocha"
+    "test": "c8 npm run test --workspaces",
```

Comment:
So I was thinking about at which point the linter is ran, and its ran as part of a post test script in each of the sub packages. For the sake of consistency, can you run c8 in that step in each package? 

Read more here -- https://medium.com/the-node-js-collection/why-and-how-to-use-eslint-in-your-project-742d0bc61ed7

Reply:
ok